### PR TITLE
sacrifice for a better code :P

### PR DESCRIPTION
**Pull request by TAImatem**
Ideally I'd also remove SetControllerButton, but that's technically part of API 🤷

### CODE REVIEW

File: OWML.ModHelper.Menus/ModPromptButton.cs [hunk 1]
```diff
@@ -21,7 +24,7 @@ public ScreenPrompt Prompt
                 _hotkeyButton.SetPrompt(value);
                 if (_prompt.GetText() != DefaultTitle)
                 {
-                    Object.Destroy(Button.GetComponentInChildren<LocalizedText>());
+                    UnityEngine.Object.Destroy(Button.GetComponentInChildren<LocalizedText>());
```
Comment (TAImatem) [inline OWML.ModHelper.Menus/ModPromptButton.cs:27]: apparently `using System` and `Object` don't like eachother

Comment (amazingalek) [reply OWML.ModHelper.Menus/ModPromptButton.cs:27]: do `using Object = UnityEngine.Object;`, then you can do `Object.Destroy`

**Comment by amazingalek** [comment]
> 
> 
> Ideally I'd also remove SetControllerButton, but that's technically part of API 🤷
It's probably only used by OWML, not by mods, so I'd say kill that too. @misternebula did you use it in your mods?

File: OWML.ModHelper.Menus/ModPromptButton.cs [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ﻿using OWML.Common.Menus;
 using OWML.ModHelper.Events;
 using UnityEngine.UI;
-using UnityEngine;
+using Object = UnityEngine.Object;
```
Comment (amazingalek) [inline OWML.ModHelper.Menus/ModPromptButton.cs:4]: oops, not needed anymore, can be simplified to `using UnityEngine;`

Comment (amazingalek) [reply OWML.ModHelper.Menus/ModPromptButton.cs:4]: Last thing ;)

**Pull request merged**